utility/src/test/java/org/oppia/android/util/logging/EventBundleCreatorTest.kt

```diff
@@ -203,7 +202,7 @@ class EventBundleCreatorTest {
     assertThat(bundle).string("memory_tier").isEqualTo("high_memory")
     assertThat(bundle).string("storage_tier").isEqualTo("high_storage")
     assertThat(bundle).string("network_type").isEqualTo("wifi")
-    assertThat(bundle).string("current_screen").isEqualTo("home_screen")
+    assertThat(bundle).string("current_screen").isEqualTo("test_screen_unspecified")
```

Comment:
Is this correct? Shouldn't it be "screen_name_unspecified"?

Reply:
done.